pkg/utils/common.go

```diff
@@ -162,3 +137,42 @@ func PrintTable(writer io.Writer, header []string, rows [][]string) {
 	table.AppendBulk(rows)
 	table.Render()
 }
+
+func StartK8sTestEnvironment() (client.Client, func(), error) {
```

Comment:
Can we put this in a different package like `testutils`?  It will be easier to catch if someone uses this in a place they shouldn't versus being mixed in with other `utils` functions that can be used within the app.

Reply:
Moved